src/java/org/apache/cassandra/tools/nodetool/ListSnapshots.java

```diff
@@ -56,6 +71,9 @@ public void execute(NodeProbe probe)
             Map<String, String> options = new HashMap<>();
             options.put("no_ttl", Boolean.toString(noTTL));
             options.put("include_ephemeral", Boolean.toString(includeEphemeral));
+            options.put("keyspace", keyspace);
```

Comment:
I would only pass these parameters as options if they are not null. The default behaviour (when we pass default null values) doesn't seem to be the right way to work with the network.

Reply:
done